tasks/gems/cloudwatchlogs.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Benchmark
+  module Gems
+    class CloudWatchLogs < Benchmark::Gem
+      def gem_name
+        'aws-sdk-cloudwatchlogs'
+      end
+
+      def gem_dir
+        "gems/#{gem_name}"
+      end
+
+      def client_module_name
+        'Aws::CloudWatchLogs'
+      end
+
+      def operation_benchmarks
+        {
+          put_log_events_small: {
+            setup: proc do |_client|
+              { log_group_name: 'log_group', log_stream_name: 'log_stream', log_events: (0...5).map do |i|
+                                                                                          { timestamp: Time.now.to_i, message: "test log event #{i}" }
+                                                                                        end }
```

Comment:
Could cut down the line length like follows? For this and others alike in this PR
```
def operation_benchmarks
  {
    put_log_events_small: {
      setup: proc do |_client|
        {
          log_group_name: 'log_group',
          log_stream_name: 'log_stream',
          log_events: (0...5).map do |i|
            { timestamp: Time.now.to_i, message: "test log event #{i}" }
          end
        }
      end
      ...
```

Reply:
Fixed up all gem files.